spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -326,7 +326,7 @@ def then_I_see_a_validation_error
     expect(page).to have_content("Title can't be blank")
     expect(page).to have_content("Slug can't be blank")
     expect(page).to have_content("Introduction can't be blank")
-    expect(page).to have_content("Meta description can't be blank")
+    expect(page).to have_content("Description can't be blank")
```

Comment:
Do steps have a description? Or has this test been updated in the wrong commit?

Reply:
@DilwoarH This looks like it's supposed to be with the Title and intro page changes.

Reply:
Also, I'm still not sure why the test has changed.